actions/websocket_actions.jsx

```diff
@@ -72,6 +72,7 @@ export function initialize() {
     WebSocketClient.initialize(connUrl);
 }
 
+window.close = close;
```

Comment:
is this intentional?

Reply:
Did you read the summary? :P

>I've included the ability to manually disconnect the websocket by opening the dev console and entering close(); to help with testing. To reconnect, just refresh the page.

I'll remove it before dev review